.landscape.yml

```diff
@@ -0,0 +1,16 @@
+test-warnings: no
```

Comment:
We didn't find `landscape` helpful at all. The builds took so long to update that they wouldn't show up until days after we merged the actual PR. If you want to keep it is fine, but I don't see it as being necessary.

Reply:
Removed .landscape.yml.